modules/apostrophe-widgets/README.md

```diff
@@ -1,14 +1,14 @@
 ## Inherits from: [apostrophe-module](../apostrophe-module/README.md)
 The base class for all modules that implement a widget, such as
-[apostrophe-rich-text-widgets](../apostrophe-rich-text-widgets/README.md),
-[apostrophe-pieces-widgets](../apostrophe-pieces-widgets/README.md) and
-[apostrophe-video-widgets](../apostrophe-video-widgets/README.md).
+[apostrophe-rich-text-widgets](/modules/apostrophe-rich-text-widgets/),
+[apostrophe-pieces-widgets](/modules/apostrophe-pieces-widgets/) and
+[apostrophe-video-widgets](/modules/apostrophe-video-widgets/).
 
-All widgets have a [schema](/tutorials/schema-guide/schema-guide.md).
+All widgets have a [schema](../../tutorials/getting-started/schema-guide.html).
```

Comment:
It looks like this link got reverted somehow.

Reply:
It should be the /tutorials/schema-guide/schema-guide.md not the .html url

Reply:
For line 7, I'm still seeing the link as "[schema](../../tutorials/getting-started/schema-guide.html)."
When it should be "[schema](/tutorials/schema-guide/schema-guide.md)"